app/templates/skeleton/modules/custom/skeleton_restful/plugins/restful/user/update/1.1/update__1_1.inc

```diff
@@ -0,0 +1,26 @@
+<?php
```

Comment:
@ordavidil Please check if you can change you resource from `update` => users and there for override the default `users` resources ( fot this to work you need to set this variable `'restful_enable_users_resource'` => FALSE);
It will require you to re-install the project.